test/cli/verify_quote.py

```diff
@@ -15,18 +15,20 @@ def get_args():
     parser = argparse.ArgumentParser()
     parser.add_argument('--url', type=str, help='the address of the ehsm_kms_server', required=True)
     parser.add_argument('--quote', type=str, help='the quote want to verify', required=True)
+    parser.add_argument('--policyId', type=str, help='the quote want to verify', required=True)
     args = parser.parse_args()
 
     base_url = args.url + "/ehsm?Action="
     return base_url, args.quote
 
-def verify_quote(base_url, quote):
+def verify_quote(base_url, quote, policyId):
```

Comment:
do not reuse the verify_quote function. add a new function verify_quote_with_policyId()

Reply:
The policyId is require parameter.